### PR TITLE
Adds a cooldown to EMP shriek

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
This PR adds a cooldown to the Changeling EMP shriek, currently 10 seconds.

## Why It's Good For The Game
Every other EMP source in the game is limited in some way, but EMP screech isn't, allowing a changeling to instantly use this ability a ton of times. It instantly kills synths. This will make it possible to survive a ling as a synth now.

## Proof Of Testing
https://github.com/user-attachments/assets/7bdccd8c-ddb3-484e-a672-2b7c0cccec25

## Changelog

:cl: ReturnToZender
balance: Changeling shrieks now have a 10 second cooldown.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/powers/shriek.dm [hunk 1]
```diff
@@ -43,16 +43,23 @@
...
+	COOLDOWN_DECLARE(dissonant_shriek_cooldown) //BUBBER EDIT: DECLARES A COOLDOWN
 
 /datum/action/changeling/dissonant_shriek/sting_action(mob/user)
 	..()
 	if(user.movement_type & VENTCRAWLING)
 		user.balloon_alert(user, "can't shriek in pipes!")
 		return FALSE
+	//BUBBER EDIT: NO PULSE IF YOU'RE ON COOLDOWN
+	if(!COOLDOWN_FINISHED(src, dissonant_shriek_cooldown))
+		user.balloon_alert(user, "Your throat is sore!")
```
Comment (nikothedude) [inline code/modules/antagonists/changeling/powers/shriek.dm:55]: balloon alert standard usually doesnt include capitalization, and usually not "your" instead just opting to say "throat is sore!"

**Comment by nikothedude** [comment]
honstly make it longer synths get fucking DECIMATED from emps, 5 secs is still enough to stunlock and kill easily

**Comment by nikothedude** [comment]
> honstly make it longer synths get fucking DECIMATED from emps, 5 secs is still enough to stunlock and kill easily

or maybe we should jsut nerf emps. again. again. again. it never ends

**Comment by StrangeWeirdKitten** [comment]
Synths no longer get absolutely decimiated by EMPs like they used to. But a cooldown is still reasonable and fine.

**Review by StrangeWeirdKitten — APPROVED** [review]

**Pull request merged**